components/server/ee/src/gitlab/gitlab-app-support.ts

```diff
@@ -61,4 +75,27 @@ export class GitLabAppSupport {
         result.unshift(...ownersRepos);
         return result;
     }
+
+    protected async getAccountAvatarUrl(project: ProjectSchema, providerHost: string): Promise<string> {
+        const owner = project.owner || project.namespace;
+        if (owner.avatar_url) {
+            const url = owner.avatar_url;
+            // Sometimes GitLab avatar URLs are relative -- ensure we always use the correct host
+            return url[0] === "/" ? `https://${providerHost}${url}` : url;
+        }
+        // If there is no avatar, generate the same default avatar that GitLab uses. Based on:
+        // - https://gitlab.com/gitlab-org/gitlab/-/blob/b2a22b6e85200ce55ab09b5c765043441b086c96/app/helpers/avatars_helper.rb#L151-161
+        // - https://gitlab.com/gitlab-org/gitlab-foss/-/blob/84b4743475246e91dc78c3f25f9b335c40be84cd/app/assets/stylesheets/startup/startup-general.scss#L1611-1631
```

Comment:
Not sure what the difference is. I'll try to find equivalent sources from the `gitlab` repo.

Reply:
> Not sure what the difference is.
The difference is that gitlab-foss does not contain any enterprise or licensed code, mostly everything inside the `ee/` top folder which is automatically dropped. Some open source orgs have a requirement of not using enterprise or licenced code and strictly use GitLab's FOSS version.

I've already linked the equivalent source on the `gitlab` repository in the code suggestion in https://github.com/gitpod-io/gitpod/pull/8824#discussion_r836723888.

Reply:
Haha, since GitHub was being slow, it took me 5 minutes to understand that you actually linked to the comment right above mine 😅 fair enough, many thanks for finding the correct source!